quickstart/speech-translation/csharp-uwp/helloworld.sln

```diff
@@ -0,0 +1,51 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.136
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "helloworld", "helloworld\helloworld.csproj", "{967862D8-3860-4876-BB22-73496C0A4057}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|ARM = Debug|ARM
+		Debug|ARM64 = Debug|ARM64
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
+		Release|ARM = Release|ARM
+		Release|ARM64 = Release|ARM64
+		Release|x64 = Release|x64
+		Release|x86 = Release|x86
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{967862D8-3860-4876-BB22-73496C0A4057}.Debug|ARM.ActiveCfg = Debug|ARM
+		{967862D8-3860-4876-BB22-73496C0A4057}.Debug|ARM.Build.0 = Debug|ARM
+		{967862D8-3860-4876-BB22-73496C0A4057}.Debug|ARM.Deploy.0 = Debug|ARM
+		{967862D8-3860-4876-BB22-73496C0A4057}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{967862D8-3860-4876-BB22-73496C0A4057}.Debug|ARM64.Build.0 = Debug|ARM64
+		{967862D8-3860-4876-BB22-73496C0A4057}.Debug|ARM64.Deploy.0 = Debug|ARM64
+		{967862D8-3860-4876-BB22-73496C0A4057}.Debug|x64.ActiveCfg = Debug|x64
+		{967862D8-3860-4876-BB22-73496C0A4057}.Debug|x64.Build.0 = Debug|x64
+		{967862D8-3860-4876-BB22-73496C0A4057}.Debug|x64.Deploy.0 = Debug|x64
+		{967862D8-3860-4876-BB22-73496C0A4057}.Debug|x86.ActiveCfg = Debug|x86
+		{967862D8-3860-4876-BB22-73496C0A4057}.Debug|x86.Build.0 = Debug|x86
+		{967862D8-3860-4876-BB22-73496C0A4057}.Debug|x86.Deploy.0 = Debug|x86
+		{967862D8-3860-4876-BB22-73496C0A4057}.Release|ARM.ActiveCfg = Release|ARM
+		{967862D8-3860-4876-BB22-73496C0A4057}.Release|ARM.Build.0 = Release|ARM
+		{967862D8-3860-4876-BB22-73496C0A4057}.Release|ARM.Deploy.0 = Release|ARM
+		{967862D8-3860-4876-BB22-73496C0A4057}.Release|ARM64.ActiveCfg = Release|ARM64
+		{967862D8-3860-4876-BB22-73496C0A4057}.Release|ARM64.Build.0 = Release|ARM64
+		{967862D8-3860-4876-BB22-73496C0A4057}.Release|ARM64.Deploy.0 = Release|ARM64
+		{967862D8-3860-4876-BB22-73496C0A4057}.Release|x64.ActiveCfg = Release|x64
+		{967862D8-3860-4876-BB22-73496C0A4057}.Release|x64.Build.0 = Release|x64
+		{967862D8-3860-4876-BB22-73496C0A4057}.Release|x64.Deploy.0 = Release|x64
+		{967862D8-3860-4876-BB22-73496C0A4057}.Release|x86.ActiveCfg = Release|x86
+		{967862D8-3860-4876-BB22-73496C0A4057}.Release|x86.Build.0 = Release|x86
```

Comment:
can you remove the ARM and ARM64 configurations in UWP solutions / projects? they aren't currently supported